src/lib/plugin.js

```diff
@@ -375,7 +377,7 @@ class FiveBellsLedger extends EventEmitter2 {
     try {
       res = yield request(Object.assign({
         method: 'get',
-        uri: creds.account,
+        uri: this.appendAuthToken(creds.account),
         json: true
       }, requestCredentials(creds)))
```

Comment:
Instead of appending the token to the URL you should pass it into the `request` call as an auth parameter (see [here](https://github.com/request/request#http-authentication)). You probably want to modify the `requestCredentials` function